styleguide/derek/global-components/footer/_footer.scss

```diff
@@ -21,6 +21,7 @@
 .footer-menu-link {
   color: $gray-light;
   font-size: 1.2rem;
+  font-weight: 500;
```

Comment:
We only have the 500 weight installed on the rackspace_spa theme, so this change won't make any sort of difference elsewhere on the site.